texar/torch/modules/classifiers/conv_classifiers.py

```diff
@@ -146,7 +152,8 @@ def default_hparams() -> Dict[str, Any]:
     def forward(self,  # type:ignore
                 input: torch.Tensor,
                 sequence_length: Union[torch.LongTensor, List[int]] = None,
-                dtype: Optional[torch.dtype] = None) \
+                dtype: Optional[torch.dtype] = None,
+                data_format: Optional[str] = None) \
```

Comment:
what's the conventional ordering of `data_format` and `dtype`? Could you check pytorch/TF APIs?

Reply:
Conventionally there's no `data_format` argument in PyTorch. There is, however, `layout` which is positioned after `dtype`. I actually think it's best if we make these arguments keyword-only, but for now I think we can keep it as is for consistency.

Reply:
I also quickly checked Tensorflow APIs. I couldn't find functions which take `data_format` and `dtype` as arguments.

Reply:
what does make these arguments keyword-only" mean?

Reply:
It means that to specify these arguments, one must write the argument name in full:

```python
def func(input, *, dtype=None):
	pass

func(tensor)                     # Correct; dtype is None
func(tensor, torch.float)        # Wrong
func(tensor, dtype=torch.float)  # Correct
func(tensor, **{"dtype": torch.float})  # Also correct
```

Also see: https://python-3-for-scientists.readthedocs.io/en/latest/python3_advanced.html